static/src/javascripts/projects/common/modules/component.js

```diff
@@ -107,7 +109,11 @@ define([
 
         this.elem = template;
         this._prerender();
-        bonzo(container)[this.manipulationType](this.elem);
+
+        fastdom.write(function () {
+            bonzo(container)[this.manipulationType](this.elem);
+        });
+
         this._ready();
```

Comment:
It would be safest to ensure that `ready()` is executed after fastdom. Fixing this at the Component level is quite risky though; it's possible the jankiness was lost because so much post ready work relied on the container existing. I'd say we look at the wider picture and work at trimming/reworking Component into something more fastdom-compatible, and easier to use.